include/mega/utils.h

```diff
@@ -222,6 +222,63 @@ struct MEGA_API MemAccess
 int mega_snprintf(char *s, size_t n, const char *format, ...);
 #endif
 
+struct MEGA_API TLVstore
+{
+private:
+    TLV_map tlv;
+
+ public:
+
+    /**
+     * @brief containerToTLVrecords Builds a TLV object with records from an encrypted container
+     * @param data Binary byte array representing the encrypted container
+     * @param datalen Length of the byte array.
+     * @param key Master key to decrypt the container
+     * @return A new TLVstore object. You take the ownership of the object.
+     */
+    static TLVstore * containerToTLVrecords(const string data, SymmCipher *key);
+
+    /**
+     * @brief containerToTLVrecords Builds a TLV object with records from a container
+     * @param data Binary byte array representing the TLV records
+     * @param datalen Length of the byte array.
+     * @return A new TLVstore object. You take the ownership of the object.
+     */
+    static TLVstore * containerToTLVrecords(const string data);
+
+
+    string TLVrecordsToContainer(SymmCipher *key, encryptionmode_t mode = AES_CCM_12_16);
+
+    string TLVrecordsToContainer();
+
+    /**
+     * @brief get Get the value for a given key
+     * @param type Type of the value (without scope nor non-historic modifiers).
+     * @return String containing the array with the value, or NULL if error.
+     */
+    string get(string type)   { return tlv.at(type); }
```

Comment:
I don't see any mention on that regard in the wiki (https://wiki.developers.mega.co.nz/SoftwareEngineering/CppCodingStyle) nor in the config file for N'SIQ CppStyle (https://github.com/meganz/sdk/blob/master/contrib/nsiq_filefilter.txt), but I'll move them to the source file :)